src/components/GlobalHeader.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import {
+  Header,
+  NavMenuButton,
+  Title,
+  GovBanner,
+  PrimaryNav,
+} from "@trussworks/react-uswds";
+
+function GlobalHeader() {
+  const [expanded, setExpanded] = useState(false);
+  const onClick = () => setExpanded((prvExpanded) => !prvExpanded);
+
+  const pages = [
+    <NavLink to="/ourTeam" key="ourTeam">
```

Comment:
Can you tell me more about why you chose to use NavLink over Link? I saw in the react router dom docs that NavLink can have different styling when active. Do you plan to use activeClassName and activeStyle?